### PR TITLE
Add isMale/isFemale

### PR DESCRIPTION
**Pull request by frozzare**
This pull request adds `isMale` and `isFeature` as described in v2.1 of package meta. Also upgrade babel and fixed some default values.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,10 +24,12 @@
     "babel-preset-env": "^1.7.0"
```
Comment (Johannestegner) [inline package.json]: I think this line should go away Right? ;)

File: src/personnummer.js [hunk 2]
```diff
@@ -148,6 +144,41 @@ module.exports = {
     return Math.floor((Date.now() - new Date(parts.century + parts.year, parts.month, day).getTime()) / 3.15576e+10);
   },
 
+  /**
+   * Check if a Swedish social security number is for a female.
+   *
+   * @param {string|number} ssn
```
Comment (Johannestegner) [inline src/personnummer.js]: I would love to see param descriptors, while It's not at all urgent and it's something I can do later if I really feel the need! :)

File: src/personnummer.js [hunk 3]
```diff
@@ -148,6 +144,41 @@ module.exports = {
...
+   *
+   * @return {boolean}
+   */
+  isMale(ssn, includeCoordinationNumber) {
+    if (!this.valid(ssn, includeCoordinationNumber)) {
+      throw new Error('Invalid swedish social security number');
+    }
+
+    const parts = getParts(ssn);
+    const genderDigit = parts['num'].substr(-1);
```
Comment (Johannestegner) [inline src/personnummer.js]: Gender !== Sex!

File: .babelrc [hunk 4]
```diff
@@ -1,11 +1,10 @@
 {
   "presets": [
-    ["env", {
+    ["@babel/preset-env", {
         "targets": {
           "browsers": ["last 1 versions", "ie >= 10"]
```
Comment (Johannestegner) [inline .babelrc]: We might want to set this to lower than last 1 versions, also, might be nice to have a node v10+ or something in there.

File: src/personnummer.js (hunk 2 above)
Comment (frozzare) [reply src/personnummer.js]: ;)

**Pull request merged**